src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -387,9 +387,9 @@ public static IEnumerable<EventSource> GetSources()
             {
                 Debug.Assert(EventListener.s_EventSources != null);
 
-                foreach (WeakReference eventSourceRef in EventListener.s_EventSources)
+                foreach (WeakReference<EventSource> eventSourceRef in EventListener.s_EventSources)
```

Comment:
Are the downlevel packages going to build with this change once it is mirrored to CoreFX?

Reply:
I believe so. Before putting this up I copied the shared folder over to corefx and successfully built all configurations. It's of course possible I missed something, though.

Reply:
I tested it again, and using the entire shared folder from corelib and doing a `build --allconfigurations` in corefx passes, so... 🤞